{"name": "test_dodawania[2-6]", "status": "failed", "statusDetails": {"message": "AssertionError: Suma dwóch takich samych składników powinna być równa 4\nassert (2 + 2) == 6", "trace": "skladnik = 2, suma = 6\n\n    @pytest.mark.parametrize(\"skladnik,suma\", [(5, 8), (2, 6)])\n    def test_dodawania(skladnik, suma):\n>       assert skladnik + skladnik == suma, \"Suma dwóch takich samych składników powinna być równa \" + str(skladnik + skladnik)\nE       AssertionError: Suma dwóch takich samych składników powinna być równa 4\nE       assert (2 + 2) == 6\n\npytest_project\\dodawanie_test.py:6: AssertionError"}, "parameters": [{"name": "skladnik", "value": "2"}, {"name": "suma", "value": "6"}], "start": 1669994098076, "stop": 1669994098076, "uuid": "7ddb2194-cfac-400a-a1dd-bc63d16ebe0d", "historyId": "60b641c3cd861cc244ed43ba474f2d5b", "testCaseId": "b49f26c57393c83da8f9d29d97e4174b", "fullName": "pytest_project.dodawanie_test#test_dodawania", "labels": [{"name": "parentSuite", "value": "pytest_project"}, {"name": "suite", "value": "dodawanie_test"}, {"name": "host", "value": "DESKTOP-OM28FAJ"}, {"name": "thread", "value": "7416-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "pytest_project.dodawanie_test"}]}